src/model/message.ts

```diff
@@ -1,32 +1,23 @@
-import { DMChannel, GroupDMChannel, Message, TextChannel } from 'discord.js'
+import { DMChannel, GroupDMChannel, Message, Structures, TextChannel } from 'discord.js'
 import { CommandoClient } from '.'
 import { Command } from './command'
 
 /**
- *  Subclass of the Discord.js Message class.
- *
- * @todo
+ * Subclass of the Discord.js Message class.
  */
 export class CommandoMessage extends Message {
```

Comment:
Im not too sure that this will work as intended.

If I try to access `msg.client` now, I think it would give me the discord.js Client object instead of the new CommandoClient, since the client property is no longer being overridden.

Reply:
No it works I tested it